rio-code/src/main/java/frc/robot/Commands/drive/AlignWithLeftReef.java

```diff
@@ -55,34 +59,41 @@ public AlignWithLeftReef() {
 
     @Override
     public void initialize() {
+        System.out.println("entered left reef align");
         switch (DashboardButtonBox.getSelectedReefBranch()) {
             case A:
                 targetReefPose = Robot.getAlliance() == Alliance.Blue ? Limelight.getTagPose(ReefTagIDs.blueReefAB) : Limelight.getTagPose(ReefTagIDs.redReefAB);
+                targetReefRotation = Robot.getAlliance() == Alliance.Blue ? new Rotation2d(Units.degreesToRadians(0)) : new Rotation2d();
                 targetReefTag = Robot.getAlliance() == Alliance.Blue ? ReefTagIDs.blueReefAB : ReefTagIDs.redReefAB;
                 break;
         
             case C:
                 targetReefPose = Robot.getAlliance() == Alliance.Blue ? Limelight.getTagPose(ReefTagIDs.blueReefCD) : Limelight.getTagPose(ReefTagIDs.redReefCD);
+                targetReefRotation = Robot.getAlliance() == Alliance.Blue ? new Rotation2d(Units.degreesToRadians(60)) : new Rotation2d();
```

Comment:
Would still like to see these in the constants file. It's not often, but sometimes you have to tweak them.